### PR TITLE
Migrate community chat to Discord

### PR DESCRIPTION
**Pull request by mavam**
This PR makes the adapations on the website and docs about our move of the community chat to Discord.

### CODE REVIEW

File: web/blog/from-slack-to-discord/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: From Slack to Discord
+description: Moving our community chat from Slack to Discord
+authors: mavam
+image: slack-to-discord.light.png
```
Comment (mavam) [inline web/blog/from-slack-to-discord/index.md:5]:
```suggestion
image: /blog/from-slack-to-discord/slack-to-discord.light.png
```

@ditya I also tried this to get the image render in the main page carousel. Not sure if it makes more sense to just remove this metadata item, or whether there was a quick fix to make it work.

Comment (dit7ya) [reply web/blog/from-slack-to-discord/index.md]: Pushed the quick fix.

Comment (mavam) [reply web/blog/from-slack-to-discord/index.md]: That works, but is the idea now that we pollute `/img` for blog post images?

File: web/blog/from-slack-to-discord/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- **Retention**: Slack's free plan has only 90 days message retention. We prefer
+  permanence of our community discussion.
+
+- **Moderation**: Discord has solid moderation tools that rely on role-based
+  access, and makes it possible adhere to our [Code of
+  Conduct](/docs/contribute/code-of-conduct) upon joining.
+
+- **Invitation**: Unlimited invite links that do not expire.
+
+- **Inclusion**: users can self-assign their preferred pronouns.
```
Comment (dit7ya) [inline web/blog/from-slack-to-discord/index.md:43]:
```suggestion
- **Inclusion**: Users can self-assign their preferred pronouns.
```

File: web/blog/from-slack-to-discord/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+We hope that the majority of our Slack users understand these concerns and
+will rejoice with us over at Discord. See you there!
```
Comment (dit7ya) [inline web/blog/from-slack-to-discord/index.md:46]:
```suggestion
will join us at Discord. See you there!
```

File: web/blog/from-slack-to-discord/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+<!--truncate-->
+
+![Slack-to-Discord](/img/slack-to-discord.light.png#gh-light-mode-only)
+![Slack-to-Discord](/img/slack-to-discord.dark.png#gh-dark-mode-only)
+
+:::info Discord Invite Link
+You can join our Discord community chat via <https://vast.io/discord>.
+:::
+
+Here are our top reasons why we switched:
```
Comment (dit7ya) [inline web/blog/from-slack-to-discord/index.md:32]:
```suggestion
Here are the top four reasons why we are switching:
```

**Review by dit7ya — APPROVED** [review]

File: web/blog/from-slack-to-discord/index.md (hunk 1 above)
Comment (dit7ya) [reply web/blog/from-slack-to-discord/index.md]: I say for now, yes. An overhaul of the docs site is planned, and I say we don't spend more time on patching it before then.

Comment (dominiklohmann) [reply web/blog/from-slack-to-discord/index.md]: Did I miss something? What's the planned overhaul for the docs site?

Comment (mavam) [reply web/blog/from-slack-to-discord/index.md]: Taking this discussion offline.

**Pull request merged**